### DISCUSSION TITLE
May be worth linking to Microsoft's Natural Voice?

### DISCUSSION THREAD

**Discussion by jeffmcjunkin** [neonbjb/tortoise-tts#112] [General]
It was a bit difficult to find the reference listed on https://nonint.com/static/tortoise_v2_examples.html for Natural Voice. 

https://techcommunity.microsoft.com/t5/ai-cognitive-services-blog/new-technical-research-is-advancing-azure-s-neural-text-to/ba-p/3499414

**Comment by neonbjb** [comment]
I'm really excited to hear more about natural speech. Microsoft seems to be staying pretty close-lipped about it, such that I can't find many examples, and the ones that I can find are all using LJSpeech. I'd like to hear some other voices and samples. Once these are available, I'll provide them directly in the comparison sheet.
